provisioning/playbooks/apache.yaml

```diff
@@ -0,0 +1,14 @@
+- hosts: apache
+  tasks:
+    - name: Install apache block
+      block:
+        - name: Install apache server in Linux systems
+          become: true
+          become_user: root
+          import_role:
+            name: ../roles/apache
+          when: ansible_os_family == "Debian" or ansible_os_family == "RedHat" or ansible_os_family == "Solaris"
+        - name: Install apache server in Windows and macOS hosts
```

Comment:
Use line break for all different tasks. Apply this to all yaml files

Reply:
Added in 5931828]